lemminx-maven/src/test/java/org/eclipse/lemminx/maven/test/LocalPluginTest.java

```diff
@@ -86,4 +89,24 @@ public void testPluginArtifactHover() throws IOException, InterruptedException,
 		assertTrue(languageService.doHover(createDOMDocument("/pom-plugin-artifact-hover.xml", languageService),
 				new Position(14, 18), new XMLHoverSettings()).getContents().getRight().getValue().contains("Maven Surefire MOJO in maven-surefire-plugin"));
 	}
+	
+	// Diagnostic related tests
+
+		@Test(timeout=30000)
+	 	public void testPluginConfigurationDiagnostics() throws IOException, InterruptedException, ExecutionException, URISyntaxException {
+			DOMDocument document = createDOMDocument("/pom-plugin-configuration-hover.xml", languageService);
```

Comment:
A new `pom-plugin-configuration-diagnostics.xml` needs to be created since the plugin goal validation is picking up the "list" goal which is not a valid goal for maven surefire plugin. Then this test should pass on CI.

Reply:
There's actually no "list" goal being used in pom-plugin-configuration-hover.xml . Some other error marker must be present...